src/file-uploader.scss

```diff
@@ -0,0 +1,32 @@
+@import "./mixins";
+@import "./new-settings";
+
+$block: #{$fd-namespace}-file-uploader;
+
+.#{$block} {
+  $fd-file-uploader-padding: 0.25rem;
+  @include fd-reset();
+
+  display: inline-flex;
+  align-items: center;
+  width: auto;
+
+  &__input.#{$fd-namespace}-input {
+    width: auto;
+    margin-right: $fd-file-uploader-padding;
+
+    @include fd-rtl() {
+      margin-left: $fd-file-uploader-padding;
```

Comment:
Does `margin-right` need to be set to "0" here for the RTL case? I'm asking, I'm not sure myself.